src/github_changelog/github.clj

```diff
@@ -22,8 +22,7 @@
 (defn make-request
   ([config] (make-request config {}))
   ([{oauth-token :token} params]
-   {:as           :json
```

Comment:
Are we sure this is not required anymore?

Reply:
According to the [source code](https://github.com/dakrone/clj-http/blob/217393258e7863514debece4eb7b23a7a3fa8bd9/src/clj_http/client.clj#L435) and the [documentation](https://github.com/dakrone/clj-http#optional-dependencies), adding `:as` with `:json` requests for a `cheshire`-based JSON coercion but that would only work if the library is actually pulled in as a dependency.